assignment-client/src/entities/EntityPriorityQueue.h

```diff
@@ -57,6 +59,7 @@ class PrioritizedEntity {
     EntityItemWeakPointer _weakEntity;
     EntityItem* _rawEntityPointer;
     float _priority;
+    bool _forceSend;
```

Comment:
Something seems wrong adding this `bool` to the `PrioritizedEntity`.  If we want to force the send, why don't we just give it a non-zero priority?  It looks like the only place that was using this new parameter was where we would add changed things to the `_sendQueue` in a signal+slot rather than in the scan.